.github/dependabot.yml

```diff
@@ -28,3 +28,6 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    ignore:
+      - dependency-name: "typescript"
+      - dependency-name: "ts-loader"
```

Comment:
Should we create an issue to update these so we don't forget? I don't think it's critical that these be updated so it'd be good to have a reminder somewhere.

Reply:
Yeah I made this issue https://github.com/apache/daffodil-vscode/issues/188 to investigate